.vscode/settings.json

```diff
@@ -1,5 +1,19 @@
 // Place your settings in this file to overwrite default and user settings.
 {
     "typescript.tsdk": "node_modules/typescript/lib",
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "workbench.colorCustomizations": {
```

Comment:
Please remove the changes to this file from this PR.

Reply:
My apologies! I didn't realize my editor modified this; will revert.